spec/Cabin/SingleSeat.vspec

```diff
@@ -133,9 +129,8 @@ HeadRestraint.Height:
   datatype: uint8
   type: actuator
   min: 0
-  max: 255
   unit: mm
-  description: Height of head restraint. 0 = Bottommost. 255 = Uppermost.
+  description: Height of head restraint. 0 = Bottommost.
```

Comment:
Another observation, we sometimes use "lowermost" and sometimes "bottommost". For me the terms seems to be semantically the same - so which one do we prefer?